extensions/linz/schema.json

```diff
@@ -16,14 +16,16 @@
           "minLength": 1
         },
         "created": {
+          "title": "Creation Time",
           "type": "string",
-          "minLength": 1,
-          "format": "date-time"
+          "format": "date-time",
+          "pattern": "(\\+00:00|Z)$"
```

Comment:
Why require a time zone specifier? [The standard already specifies that the datetime needs to be UTC](https://github.com/radiantearth/stac-spec/blob/f81324f712a608d4a319fd0a8738193a43ef1d51/item-spec/common-metadata.md#date-and-time).

Reply:
https://github.com/linz/geostore/issues/978#issuecomment-923451562

Reply:
https://linz.slack.com/archives/CN7F6HP2Q/p1632178810030700

don't think so. These are new fields meant for the collection metadata- those dates eist in the parent schema only for Items i beilieve.